install_config/persistent_storage/persistent_storage_fibre_channel.adoc

```diff
@@ -51,11 +51,14 @@ spec:
   accessModes:
     - ReadWriteOnce
   fc:
-    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <1>
+    wwids: [`scsi-3600508b400105e210000900000490000 -> ../../sda`, \
+    `scsi-SSEAGATE_ST373453LW_3HW1RHM6 -> ../../sda`] <1>
+    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <2>
     lun: 2
```

Comment:
1. FC pv can use targetwwns and lun or wwids. It suggests to use wwids 
2. wwids in the doc should like "scsi-3600508b400105e210000900000490000"